src/gui_common/tooltip/microbe_editor/SelectionMenuToolTip.cs

```diff
@@ -335,6 +278,52 @@ public void OnHide()
         Hide();
     }
 
+    /// <summary>
+    ///   Searches the processes description string for compound name and input action
+    ///   "keys" and turns them into a BBCode string with matching values.
+    /// </summary>
+    private string ParseProcessesDescription()
+    {
+        if (string.IsNullOrEmpty(ProcessesDescription))
+            return string.Empty;
+
+        var inputEvents = Settings.GetCurrentlyAppliedControls();
+
+        var result = Regex.Replace(TranslationServer.Translate(ProcessesDescription), @"\[(.*?)\]", found =>
+        {
+            var parsed = string.Empty;
+
+            var value = found.Groups[1].Value;
+
+            // Parse compound names
+            if (SimulationParameters.Instance.DoesCompoundExist(value))
+            {
+                var compound = SimulationParameters.Instance.GetCompound(value);
+
+                parsed = $"[b]{compound.Name}[/b] [font=res://src/gui_common/fonts/" +
+                    $"BBCode-Image-VerticalCenterAlign-3.tres] [img=20]{compound.IconPath}[/img][/font]";
+            }
+
+            // Parse input actions
+            if (InputMap.HasAction(value))
+            {
+                var events = inputEvents.Data[value];
+
+                for (int i = 0; i < events.Count; i++)
```

Comment:
I think only the primary binding should be shown here to reduce clutter in case someone adds like 5 key bindings for an action.